### ISSUE TITLE
Group Open Function doesn't get 'group component' argument correctly

### ISSUE THREAD

**Issue by xogoc** [olifolkerd/tabulator#1391]
Version 3.5
So in this code from documentation:
```
groupStartOpen:function(value, count, data, group){
    //value - the value all members of this group share
    //count - the number of rows in this group
    //data - an array of all the row data objects in this group
    //group - the group component for the group

    return count > 3; //all groups with more than three rows start open, any with three or less start closed
},
```
**group** will be another **count** and not the group component.

**Comment by olifolkerd** [comment]
It shouldn't get the group element. It is passed the group component object. To get the element you need to call the gerComponent function on the component passed into the function

**Comment by olifolkerd** [comment]
Ahh with you now. That issue has been fixed in version 4.0 which is currently in public beta. It should be released in two weeks time

**Comment by xogoc** [comment]
Thanks. Tabulator is awesome!
